packages/core/src/features/st-var.ts

```diff
@@ -28,10 +27,18 @@ export interface VarSymbol {
     node: postcss.Node;
 }
 
+export type Input = Box<string, Input | Record<string, Input | string> | Array<Input | string>>;
+
 export interface ComputedStVar {
     value: RuntimeStVar;
     diagnostics: Diagnostics;
-    input?: any;
+    input: Input;
+}
+
+export interface FlatComputedStVar {
+    value: string;
+    symbol: string;
+    path: string[];
```

Comment:
I find `symbol` unclear, why is it needed?

Reply:
We will call `symbol` -> `jsPath`